website/content/docs/commands/operator/snapshot/restore.mdx

```diff
@@ -15,6 +15,9 @@ designed to handle server failures during a restore. This command is primarily
 intended to be used when recovering from a disaster, restoring into a fresh
 cluster of Nomad servers.
 
+This command only restores the Raft snapshot. If you are recovering a cluster,
+you will also need to [restore the keyring][] onto at least one server.
```

Comment:
```suggestion
~> **Warning:** This command only restores the Raft snapshot. If you are recovering a cluster,
you will also need to [restore the keyring][] onto at least one server.
```

> but for typical cloud workflows where the whole host is replaced, it's an important warning

This is a good point, and I haven't thought of this before. So maybe it's worth highlighting these more?